index.js

```diff
@@ -161,16 +161,43 @@ module.exports = class TrailsApp extends events.EventEmitter {
   }
 
   /**
+   * Extend the once emiter reader for accept multi valid events
+   */
+  onceAny (events, handler) {
+    if (!events)
+      return
+    if (!(events instanceof Array))
+      events = [events]
+    const self = this
```

Comment:
Maybe this can be simplified by using ES2015 and arrow functions, to prevent using this kind of variable